activelearning/dataset/ocp.py

```diff
@@ -74,7 +74,7 @@ def __getitem__(self, key):
 
         x, y = self.preprocess(x.to(self.float).squeeze(), y.to(self.float).squeeze())
         if self.return_target:
-            return x, y
+            return x, y * -1  # turn into maximization problem
```

Comment:
I would document this a bit more clearly. For example at the head of the script.